### PR TITLE
Add registration blog post for Developer Conference 2024

### PR DESCRIPTION
**Pull request by coffrep**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Adding the registration blog post for Developer Conference 2024
| Fixed ticket? | None

### CODE REVIEW

**Comment by coffrep** [comment]
Hi @kpodemski here is the registration blog post for the blog. I've created a new folder for october visuals to host the image.

File: RegisterPrestaShopDeveloperConference2024.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title: Register Now for the PrestaShop Developer Conference 2024
+subtitle: CONFERENCE REGISTRATION IS NOW OPEN
```
Comment (kpodemski) [inline RegisterPrestaShopDeveloperConference2024.md]: There is no need to use capitalized letters

File: RegisterPrestaShopDeveloperConference2024.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+date: 2024-10-01 09:55:00
```
Comment (kpodemski) [inline RegisterPrestaShopDeveloperConference2024.md]:
```suggestion
date: 2024-10-01
```

no need to use timestamp unless we publish two articles a day

File: RegisterPrestaShopDeveloperConference2024.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+icon: icon-leaf
+tags:
+- news
+- event
+- community
+---
+
+The world of ecommerce is constantly evolving, and staying on top of the latest trends and innovations is essential for developers and businesses. That’s why we are thrilled to [invite you](https://events.prestashop.com/prestashop-developer-conference/fr) to the second edition of the PrestaShop Developer Conference, taking place on November 6, 2024, in Paris. 
+
+![Register_Developer_Conference](/assets/images/2024/10/RegisterDeveloperConference24.jpeg)
```
Comment (kpodemski) [inline RegisterPrestaShopDeveloperConference2024.md]: I think this image would be better: https://pbs.twimg.com/media/GYywfD5XAAMDSFe?format=jpg&name=medium

File: RegisterPrestaShopDeveloperConference2024.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+**Why Attend the Conference?**
```
Comment (kpodemski) [inline RegisterPrestaShopDeveloperConference2024.md]:
```suggestion
## Why Attend the Conference?
```

File: RegisterPrestaShopDeveloperConference2024.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+The PrestaShop Developer Conference is more than just a series of presentations - it’s a space where passionate developers come together to exchange ideas, learn from one another, and engage with some of the leading voices in the PrestaShop community. This year’s conference will feature live demonstrations and in-depth technical sessions aimed at enhancing your understanding of PrestaShop and its ecosystem.
+
+You will have the opportunity to dive into module and theme development, exploring the latest techniques to improve functionality and design. Additionally, the event will introduce you to PrestaShop's newest features, empowering developers to build even more dynamic and efficient ecommerce solutions. You’ll gain exclusive insights into the tools and resources available for developers.
+
+Check the full agenda on the [event’s website](https://events.prestashop.com/prestashop-developer-conference/en).
+
+**Register today**
```
Comment (kpodemski) [inline RegisterPrestaShopDeveloperConference2024.md]:
```suggestion
## Register today
```

File: RegisterPrestaShopDeveloperConference2024.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
+---
```
Comment (kpodemski) [inline RegisterPrestaShopDeveloperConference2024.md]: This file is in the wrong place, it should be inside `content/news/posts/2024/` and it's name should be:

register-for-the-prestashop-developer-conference-2024.md

**Review by kpodemski — CHANGES_REQUESTED** [review]
Thanks @coffrep, I'm leaving some comments

**Comment by coffrep** [comment]
Hi @kpodemski thanks so much for the comments on the document, I've made the edits on my end on my branch. Let me know if you see any other changes to make - thanks a lot.

**Review by kpodemski — APPROVED** [review]

**Pull request merged**